README.md

```diff
@@ -197,24 +204,24 @@ Meant to be passed as an option to [fromError](#fromerror), [fromZodIssue](#from
 
 ##### createMessageBuilder Options
 
-| Name                 |         Type          | Description                                                                                                                                                                                                                                                                                                                                  |
-| -------------------- | :-------------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `maxIssuesInMessage` |       `number`        | Max issues to include in user-friendly message (optional, defaults to `99`)                                                                                                                                                                                                                                                                  |
-| `issueSeparator`     |       `string`        | Used to concatenate issues in user-friendly message (optional, defaults to `";"`)                                                                                                                                                                                                                                                            |
-| `prefix`             | `string \| undefined` | Prefix to use in user-friendly message (optional, defaults to `"Validation error"`). Pass `undefined` to disable prefix completely.                                                                                                                                                                                                          |
-| `prefixSeparator`    |       `string`        | Used to concatenate prefix with rest of the user-friendly message (optional, defaults to `": "`). Not used when `prefix` is `undefined`.                                                                                                                                                                                                     |
-| `error`              |  `ErrorMap \| false`  | Accepts an `errorMap` to format individual issues into user-friendly error messages (optional). When set to `false` it will return the message as formatted by `zod`. Note that this is an optional property and if not provided, the default zod-validation-error error map will be used. You can use your own custom errorMap if you want. |
```

Comment:
dev: drop `error` - this is an antipattern that causes custom error messages to be overwritten.